Makefile

```diff
@@ -51,7 +51,9 @@ testnsxv:
 # any common errors.
 vet:
 	@echo "==> Running Go Vet"
-	@cd govcd && go vet ; if [ $$? -ne 0 ] ; then echo "vet error!" ; exit 1 ; fi && cd -
+	@cd govcd && go vet ; if [ $$? -ne 0 ] ; then echo "vet error!" ; exit 1 ; fi
```

Comment:
I suggest using the canonical `go vet ./...` instead of these three lines.
In addition to being easier to maintain, it also covers the `./samples` directory, which would be left out by the current commands.

Reply:
Changed.